product_docs/docs/migration_toolkit/55/06_building_toolkit.properties_file.mdx

```diff
@@ -297,29 +297,53 @@ Migration Toolkit reads connection specifications for the source database from t
 -   The name of a privileged user
 -   The password associated with the specified user
 
-If you're connecting to a SQL Server database, `SRC_DB_URL` takes the form of a JTDS URL. For example:
+If you're connecting to a SQL Server database, `SRC_DB_URL` takes the form of a JTDS URL or Microsoft JDBC URL, as follows:
```

Comment:
```suggestion
If you're connecting to a SQL Server database, `SRC_DB_URL` takes the form of a [JTDS URL](#jtds-url) or [Microsoft JDBC URL](microsoft-jdbc-url).
```

Reply:
Is a # required before "microsoft-jdbc-url" in the hyperlink definition?

Reply:
Fixed.